config/site_settings.yml

```diff
@@ -619,6 +619,10 @@ login:
     default: 1440
     min: 1
     max: 175200
+  experimental_full_page_login:
+    default: false
+    client: true
```

Comment:
I would make this setting hidden initially, so we can focus on internal testing first.